common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1368,40 +1385,46 @@ object CelebornConf extends Logging {
       .createWithDefault(4)
 
   val PUSH_MAX_REVIVE_TIMES: ConfigEntry[Int] =
-    buildConf("celeborn.push.revive.maxRetries")
+    buildConf("celeborn.client.push.revive.maxRetries")
       .categories("client")
       .version("0.3.0")
       .doc("Max retry times for reviving when celeborn push data failed.")
       .intConf
       .createWithDefault(5)
 
   val TEST_RETRY_REVIVE: ConfigEntry[Boolean] =
-    buildConf("celeborn.test.retryRevive")
+    buildConf("celeborn.test.client.retryRevive")
+      .withAlternative("celeborn.test.retryRevive")
       .internal
-      .categories("client", "test")
+      .categories("test", "client")
       .doc("Fail push data and request for test")
       .version("0.2.0")
       .booleanConf
       .createWithDefault(false)
 
   val PUSH_TIMEOUT_CHECK_INTERVAL: ConfigEntry[Long] =
-    buildConf("celeborn.push.timeoutCheck.interval")
-      .categories("client", "worker")
-      .doc("Interval for checking push data timeout.")
+    buildConf("celeborn.<module>.push.timeoutCheck.interval")
+      .categories("network")
+      .doc("Interval for checking push data timeout. " +
+        s"If setting `module` to `${TransportModuleConstants.DATA_MODULE}`, it works for shuffle client push data to worker." +
+        s"If setting `module` to `${TransportModuleConstants.REPLICATE_MODULE}`, it works for worker replicate data to peer worker.")
```

Comment:
and should be configured in worker side.

Reply:
Done